File: Client/Client/Models/Place.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Client.Models
{
	[Serializable]
	public class Place
	{
		public long id { get; set; }
		public double longitude { get; set; }
		public double latitude { get; set; }
		public string title { get; set; }
		public string description { get; set; }
		public List<Review> reviews { get; set; }
		public UserData addedBy { get; set; }

		public Place()
		{
			reviews = new List<Review>();
		}

		public Place(double longitude, double latitude, string title, string description)
		{
			this.longitude = longitude;
			this.latitude = latitude;
			this.title = title;
			this.description = description;
			reviews = new List<Review>();
		}

		public void AddReview(Review review)
		{
			reviews.Add(review);
		}

		public double GetRating()
		{
			long score = 0;
			foreach (Review item in reviews)
			{
				score += item.rating;
			}
			return (double)score / reviews.Count;
		}

		public List<Review> GetReviews()
		{
			return reviews;
		}
	}
}

File: ModeratorServer/src/main/java/dk/groupfive/ModeratorServer/model/objects/Place.java
package dk.groupfive.ModeratorServer.model.objects;

import java.io.Serializable;
import java.util.ArrayList;

public class Place implements Serializable {
    private long id;
    private double latitude;
    private double longitude;
    private String title;
    private String description;
    private ArrayList<Review> reviews;
    private UserData addedBy;

    public Place() {

    }

    public Place(long id, double latitude, double longitude, String title, String description) {
        this.id = id;
        this.latitude = latitude;
        this.longitude = longitude;
        this.title = title;
        this.description = description;
    }

    public Place(double latitude, double longitude, String title, String description) {
        this.latitude = latitude;
        this.longitude = longitude;
        this.title = title;
        this.description = description;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public ArrayList<Review> getReviews() {
        return reviews;
    }

    public void setReviews(ArrayList<Review> reviews) {
        this.reviews = reviews;
    }

    public void addReview(Review reviewItem)
    {
        reviews.add(reviewItem);
    }

    public UserData getAddedBy() {
        return addedBy;
    }

    public void setAddedBy(UserData addedBy) {
        this.addedBy = addedBy;
    }

    @Override
    public String toString() {
        return "Place{" +
                "id=" + id +
                ", latitude=" + latitude +
                ", longitude=" + longitude +
                ", title='" + title + '\'' +
                ", description='" + description + '\'' +
                '}';
    }
}
